addons-dev/aldryn-cms/setup.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name="aldryn-cms",
+    version="3.1.2",
+    install_requires=(
+        'django-cms==3.1.2',
+        # common
+        # TODO: mostly to be split out into other packages
+        'django-compressor',
+        'django-health-check',
+        'YURL',
+        'South',
+        'requests',
+        'Pillow',
+        'lxml',
+        'django-treebeard',
+        'django-simple-captcha',
+        'BeautifulSoup',
+        'subprocess32',
+        'django-parler',
+        'django-robots',
+        'aldryn-boilerplates',
+        'aldryn-sites',
+    ),
```

Comment:
Stricter versions will be added here. But they can't be too strict, because dependency checking will catch impossible dependencies.